### ISSUE TITLE
User Dashboard Page

### ISSUE THREAD

**Issue by rui-han** [hrcn/pushkin-sitetemplates-default#3]
I'm building the user dashboard page. But I'm still quite confused about that how should I display the experiments cards. What contents should be concluded in the card, like experiment time, experiment results?

**Comment by rui-han** [comment]
Also the user profiles. The users are allowed to edit their name, age, gender. Should I create a react form in the dashboard page, using axios and knex to communicate with the database to store and read the data?

**Comment by jkhartshorne** [comment]
> Also the user profiles. The users are allowed to edit their name, age, gender. Should I create a react form in the dashboard page, using axios and knex to communicate with the database to store and read the data?

Yes. We need a centralized database table that contains demographic information about users. I think all demographic information should go into a JSON. You can now have JSONs natively in Postgres. Or you can just stringify them. 

The reason to use a JSON is that way we don't have to decide in advance what all of the keys will be.
